src/index_set_accessontrol_alert.js

```diff
@@ -0,0 +1,69 @@
+exports.handler = function(event,context, callback) {
```

Comment:
Correct the filename, it should be "index_set_accesscontrol_alert.js" or "index_alert_console_sign_in.js"